src/components/ResultsBody/PaginatedList/usePaginatedList.js

```diff
@@ -8,6 +9,8 @@ export const usePaginatedList = (limit) => {
   const [numberOfAllPages, setNumberOfAllPages] = useState(null);
   const [page, setPage] = useState(1);
 
+  const contextSetIsError = useContext(Context)[1];
```

Comment:
You are able to use destructuring in the useState, but with useContext you are using weird indexes. Please not that useState is also returning a 2 elements array, but you are restructuring it in elegant way. You can do the same with the context.

Reply:
done